apps/web/app/s/[surveyId]/components/LinkSurvey.tsx

```diff
@@ -39,8 +39,10 @@ export default function LinkSurvey({
   const responseId = singleUseResponse?.id;
   const searchParams = useSearchParams();
   const isPreview = searchParams?.get("preview") === "true";
+  const source = searchParams?.get("source");
   // pass in the responseId if the survey is a single use survey, ensures survey state is updated with the responseId
   const [surveyState, setSurveyState] = useState(new SurveyState(survey.id, singleUseId, responseId));
+  const [metadata, setMetadata] = useState<string>("");
```

Comment:
please use a different variable name, "metadata" is too generic because the submission has all kinds of meta data like device, language, etc.

Reply:
Got it!